pkgs/development/node-packages/aliases.nix

```diff
@@ -44,6 +44,7 @@ mapAliases {
   "@nestjs/cli" = pkgs.nest-cli; # Added 2023-05-06
   eslint_d = pkgs.eslint_d; # Added 2023-05-26
   manta = pkgs.node-manta; # Added 2023-05-06
+  semantic-release = pkgs.semantic-release; # Added 2022-06-09
```

Comment:
I don't think it's necessary to include the package in the `aliases.nix` file, if it was never in the `nodePackages` attribute set in the first place. It should be there just for backwards compatibility and possibly deprecation at some point in the future.

Reply:
@ehllie thank you for the explanation. I removed the package from `aliases.nix`